test/src/module/integration/allTest.c

```diff
@@ -288,6 +291,9 @@ testRun(void)
             // Stop the cluster
             HRN_HOST_PG_STOP(pg1);
 
+            // Set unique spool path
+            UPDATE_SPOOL_PATH();
+
```

Comment:
I think it would be simpler to build this logic into TEST_HOST_BR() when command is CFGCMD_RESTORE. That will make it less likely to be forgotten.